### PR TITLE
cps edits

### PR DESCRIPTION
**Pull request by smichr**
I'm not sure about whether to use \verb or \texttt for reference to SymPy functions. I see mixed usage throughout the document. 

Before committing these changes, it would be nice if someone would check them over.

Note especially that I added reference to some of the other solvers in the solvers module.

### CODE REVIEW

**Comment by aktech** [comment]
Hi @smichr great to see you here :)

**Comment by smichr** [comment]
Thanks! Appreciate the help from you and @isuruf and the heads up from @certik and @asmeurer .

**Comment by asmeurer** [comment]
@smichr, if you have a chance:
- Could you rebase this against the `reviews` branch. That is the most up-to-date version of the paper. Unfortunately, that may require a lot of merge conflict resolution. 
- Can you add your author information to the authors.json.

File: diophantine.tex [hunk 1]
```diff
@@ -49,5 +50,9 @@
 42*t1**2 + 42*t2**2 + 42*t3**2 + 42*t4**2 + 42*t5**2)])
 
 >>> diophantine(a**2 + b**2 + c**2 + d**2 + e**2 + f**2 - 112)
-set([(8, 4, 4, 4, 0, 0)])
+set([(2, 3, 3, 4, 5, 7), (2, 2, 4, 4, 6, 6), (0, 3, 3, 3, 6, 7), (0, 2, 3, 5, 5,
```
Comment (asmeurer) [inline diophantine.tex:53]: All the examples need to work against SymPy 1.0. Is there a better diophantine example we can use here that works in SymPy 1.0?

Comment (smichr) [inline diophantine.tex:53]: Oh, I see. Updates to 1.0 are not included. My edits to the diophantine doc will have to change a bit, too.

**Comment by certik** [comment]
Aaron, did you merge this, or what is the status? It's hard to see from my
phone, it looks like you just closed it.

**Comment by certik** [comment]
Looks like the commits somehow got into master, so all is good.

**Comment by asmeurer** [comment]
It was against the wrong branch. @smichr pushed the changes into `reviews`.

**Comment by certik** [comment]
Ok, thanks.

**Pull request closed without merging**